install-dask.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# Copyright (c) 2023, NVIDIA CORPORATION.
+
+# Abort script on first error, undef vars are errors, propagate failures in pipelines
+set -eu -o pipefail
+
+# Always build and install dask and distributed from main
+echo "Building/installing dask and distributed from main, using $(python --version), $(pip --version)"
+pip install "git+https://github.com/dask/distributed.git@main" --upgrade
+pip install "git+https://github.com/dask/dask.git@main" --upgrade
```

Comment:
Curious on why we are doing this ?

Reply:
At one point I thought we needed to install dask from main for proper UCX support, but I could be mistaken.  If we don't need this, then I can remove the entire script.